screed/fadbm.py

```diff
@@ -17,3 +20,4 @@
     read_fasta_sequences(filename)
 
     print("Database saved in %s%s" % (sys.argv[1], DBConstants.fileExtension))
+    exit(0)
```

Comment:
We had issues before because the tests were using [execfile][1] for running scripts, and the return code (if not set explicitly) was `None` instead of `0`. We don't have the issue anymore, but doesn't hurt to be explicit.

Real answer: I just made `screed/fadbm.py` and `screed/fqdbm.py` consistent, the later had the `exit(0)` and the former didn't =]

[1]: https://docs.python.org/2/library/functions.html#execfile